zonemaster/zonemaster
# Specification of test zones and test zones for NAMESERVER15

## Purpose

* Specification of test zones for Nameserver15
* Test zones are included.
* The address plan for the test environment has been rewritten in a new format.
* Instructions for compiling CoreDNS has been updated to gain support for CHAOS queries.

## Context

Matches NAMESERVER15 as updated in #1199. Test zones covers all message tags in various scenarios.


## How to test this PR

* Review the definitions of the scenarios.
* Create a server with the test zone environment. Make sure to follow the updated instructions to get support for CHAOS queries required by this test case. 
* Verify the test zones with dig. Optionally verify against Nameserver15 before update with the help of `zonemaster-cli`on the test server.
* Match against updated Nameserver15 with the help of `zonemaster-cli`.
